app/models/generic_object.rb

```diff
@@ -1,4 +1,6 @@
 class GenericObject < ApplicationRecord
+  include CustomActionsMixin
```

Comment:
GenericObjects get their custom buttons through the `generic_object_definition` ([app/models/generic_object.rb#L12-L18](https://github.com/ManageIQ/manageiq/blob/master/app/models/generic_object.rb#L12-L18)) model which already includes this mixin.  This is not needed.